src/index.js

```diff
@@ -0,0 +1,7 @@
+exports.say = function (name) {
+    if(name!=null){
+        return('Hello '+name+'!');
+    }
+    console.log('Hello world!');
```

Comment:
This console log is useful for debugging but useless in production. Please remove all useless console logs before submitting PRs 😉